src/chunk/render/tilemap-square.vert

```diff
@@ -45,7 +45,8 @@ void main() {
         vec2(sprite_rect.end.x, sprite_rect.begin.y),
         sprite_rect.end
     );
-    v_Uv = floor(atlas_positions[gl_VertexIndex % 4] + vec2(0.01, 0.01)) / AtlasSize;
```

Comment:
I was unsure what the 0.01 was doing here. perhaps another attempt at fixing this? The floor is still a good thing to have.